bin/Makefile

```diff
@@ -17,7 +17,7 @@
 all: s2nc s2nd
 include ../s2n.mk
 
-LDFLAGS += -L../lib/ -ls2n ${LIBS}
```

Comment:
Doh, didn't realize this was needed when libcrypto.a isn't available here: https://github.com/awslabs/s2n/blob/master/lib/Makefile#L16